saleor/static/dashboard-next/translations/components/TranslationFields/TranslationFieldsLong.tsx

```diff
@@ -0,0 +1,57 @@
+import TextField from "@material-ui/core/TextField";
+import Typography from "@material-ui/core/Typography";
+import * as React from "react";
+
+import { ConfirmButtonTransitionState } from "../../../components/ConfirmButton";
+import Form from "../../../components/Form";
+import i18n from "../../../i18n";
+import TranslationFieldsSave from "./TranslationFieldsSave";
+
+interface TranslationFieldsLongProps {
+  disabled: boolean;
+  edit: boolean;
+  initial: string;
+  saveButtonState: ConfirmButtonTransitionState;
+  onSubmit: (data: string) => void;
+}
+
+const TranslationFieldsLong: React.FC<TranslationFieldsLongProps> = ({
+  disabled,
+  edit,
+  initial,
+  saveButtonState,
+  onSubmit
+}) =>
+  edit ? (
+    <Form
+      initial={{ translation: initial }}
+      onSubmit={data => onSubmit(data.translation)}
+    >
+      {({ change, data, reset, submit }) => (
+        <div>
+          <TextField
+            disabled={disabled}
+            fullWidth
+            multiline
+            // label={i18n.t("Translation")}
```

Comment:
All over PR are places with commented lines. Shouldn't those be removed?

Reply:
Most of them are generated, so it's kinda pointless to remove them. 
There are a few places with commented buttons or something, but they are politely waiting for the next PR that does some tinkering there.